.gitignore

```diff
@@ -5,6 +5,7 @@
 .tox
 .venv
 .vscode/
+**/__pycache__/
```

Comment:
Do we need these `.gitignore` changes, or does IDE automatically generate them?

Reply:
At first, I also wondered if this was unnecessary, but then I noticed that my local repo shows untracked files like 

> tests/__pycache__/
> tests/countries/__pycache__/
> ...

So I think it will be useful to add this line.